src/intersection-observer.js

```diff
@@ -219,6 +219,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.iframe_) {
+      return;
```

Comment:
I don't find where `this.boundFlush` and `this.pendingChanges` are removed. But IntersectionObserver itself can be set to null in `unLayoutCallback()`.